lib/task_helpers.rb

```diff
@@ -54,12 +54,11 @@ def self.catchup_index(start_time, index_name, batch_size=100, sleep_time=0)
     def self.create_index(name=nil)
       name ||= "#{Content::ES_INDEX_NAME}_#{Time.now.strftime('%Y%m%d%H%M%S%L')}"
 
-      mappings = {}
-      [Comment, CommentThread].each do |model|
-        mappings.merge! model.mappings.to_hash
-      end
+      Elasticsearch::Model.client.indices.create(index: name)
 
-      Elasticsearch::Model.client.indices.create(index: name, body: {mappings: mappings})
+      [CommentThread, Comment].each do |model|
```

Comment:
@maxrothman: Maybe you can output the equivalent of `curl 'es.edx:9200/_mapping' from Production so we can see the mappings and see which comes first?  Assuming the order matches the order here when tests are passing (see output from Travis tests on this branch), then we need to decide whether this is good enough?

Even then, we are back to the original question of whether to keep these with the create_index call and how exactly migrations would be dealt with.  Do we need to figure that out as part of this work, given that we aren't doing any migration that requires a change in the mappings?